### PR TITLE
Adding backup and restore JSONs.

### PR DESCRIPTION
**Pull request by branupama**
This commit adds JSONs having information for backup and restore functionality for rainier 2U system.

And also removed this info from config JSON.

### CODE REVIEW

File: configuration/configuration.hpp [hunk 1]
```diff
@@ -21,5 +21,9 @@ types::SystemTypeMap systemType{
     {"50003000",
      {"50003000_v2", {{"000A", ""}, {"000B", ""}, {"000C", ""}, {"0014", ""}}}},
     {"50004000", {"50004000", {}}}};
+
+std::unordered_map<std::string, std::string> systemSpecificBackupMap{
+    {"50001001", "backup_map_50001001.json"},
+    {"50001001_v2", "backup_map_50001001_v2.json"}};
```
Comment (SunnySrivastava1984) [inline configuration/configuration.hpp:27]: Why we need two back up maps for these two. They are just duplicate.

File: configuration/configuration.hpp [hunk 2]
```diff
@@ -21,5 +21,9 @@ types::SystemTypeMap systemType{
     {"50003000",
      {"50003000_v2", {{"000A", ""}, {"000B", ""}, {"000C", ""}, {"0014", ""}}}},
     {"50004000", {"50004000", {}}}};
+
+std::unordered_map<std::string, std::string> systemSpecificBackupMap{
+    {"50001001", "backup_map_50001001.json"},
```
Comment (SunnySrivastava1984) [inline configuration/configuration.hpp:26]: we need to come up with better names.

**Review by SunnySrivastava1984 — CHANGES_REQUESTED** [review]

File: configuration/configuration.hpp (hunk 2 above)
Comment (branupama) [reply configuration/configuration.hpp:26]: updated.

File: configuration/configuration.hpp (hunk 1 above)
Comment (branupama) [reply configuration/configuration.hpp:27]: removed json for v2, will use same json for v2 system.

**Review by SunnySrivastava1984 — APPROVED** [review]

**Review by jinuthomas — APPROVED** [review]

**Pull request merged**